### PR TITLE
Fix long polling issue for play 2.6

### PR DESCRIPTION
**Pull request by manipah**

### CODE REVIEW

**Comment by manipah** [comment]
@jfarcand hi this is the pr we talked about in the email, please have a look :)

File: module/pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <groupId>org.atmosphere</groupId>
     <artifactId>atmosphere-play</artifactId>
     <name>Atmosphere Play Module</name>
-    <version>2.4.1-SNAPSHOT</version>
+    <version>2.4.2</version>
```
Comment (jfarcand) [inline module/pom.xml:13]: Do not change the version.It will be automatically bumped when the official release happens

File: pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <groupId>org.atmosphere</groupId>
     <artifactId>play-project</artifactId>
     <name>Atmosphere Play Project</name>
-    <version>2.4.1-SNAPSHOT</version>
+    <version>2.4.2</version>
```
Comment (jfarcand) [inline pom.xml:12]: Same here

**Review by jfarcand — CHANGES_REQUESTED** [review]

File: module/pom.xml (hunk 1 above)
Comment (manipah) [inline module/pom.xml:13]:
```suggestion
    <version>2.4.1-SNAPSHOT<</version>
```

File: pom.xml (hunk 2 above)
Comment (manipah) [inline pom.xml:12]:
```suggestion
    <version>2.4.1-SNAPSHOT</version>
```

File: module/pom.xml [hunk 3]
```diff
@@ -10,7 +10,7 @@
     <groupId>org.atmosphere</groupId>
     <artifactId>atmosphere-play</artifactId>
     <name>Atmosphere Play Module</name>
-    <version>2.4.1-SNAPSHOT</version>
+    <version>2.4.1-SNAPSHOT<</version>
```
Comment (manipah) [inline module/pom.xml:13]:
```suggestion
    <version>2.4.1-SNAPSHOT</version>
```

**Review by jfarcand — APPROVED** [review]

**Pull request merged**